Sources/muterCore/MuterRunSteps/DiscoverSourceFiles.swift

```diff
@@ -74,6 +77,7 @@ private extension DiscoverSourceFiles {
                         root: rootPath
                     )
                 )
+                .map { String($0.dropFirst(2)) }
```

Comment:
interesting, why is this needed?

Reply:
some line above I am doing ` .map { "./" + $0 }`. This line removes the `./`.